netbox/ipam/models/ip.py

```diff
@@ -892,7 +892,7 @@ def family(self):
     def is_oob_ip(self):
         if self.assigned_object:
             parent = getattr(self.assigned_object, 'parent_object', None)
-            if parent.oob_ip_id == self.pk:
+            if parent.__class__.__name__ == "Device" and parent.oob_ip_id == self.pk:
```

Comment:
I think it'd be much cleaner to use hasattr(parent, oob_ip_id)

Reply:
Also, when checking the type of an object, a string comparison is never the right option. Use isinstance.

Reply:
adjusted to hasattr. it worked as expected locally